packages/contracts/contracts/optimistic-ethereum/ovm/ExecutionManager.sol

```diff
@@ -678,10 +661,7 @@ contract ExecutionManager {
         // Get the runtime bytecode
         bytes memory codeContractBytecode = stateManager.getCodeContractBytecode(codeContractAddress);
         // Safety check the runtime bytecode
-        if (!safetyChecker.isBytecodeSafe(codeContractBytecode)) {
-            // Contract runtime bytecode is not safe.
-            return false;
-        }
+        require(safetyChecker.isBytecodeSafe(codeContractBytecode), "Contract runtime (deployed) bytecode is not safe");
```

Comment:
Will we still need to return the address it _would have been_ deployed to in this case? If so, this will not do that, right?

Reply:
see above^ 😊